apps/workers/db-check/main.go

```diff
@@ -0,0 +1,86 @@
+package main
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/jmoiron/sqlx"
+	tools "github.com/pcelvng/task-tools"
+	"github.com/pcelvng/task-tools/bootstrap"
+	"github.com/pcelvng/task-tools/db"
+	"github.com/pcelvng/task/bus"
+)
+
+const (
+	taskType    = "task.dbcheck"
+	description = `
+Runs record count checks against a given table's data to verify that the data is updated as expected
+
+uri params:
+table      - required table name to validate
+date_field - required field name that holds the date value
+date1      - 1st date value to check for records
+date2      - 2nd date to compare row counts between date1 and date2
+offset     - if date2 is not given, this is the offset duration back from date1 to get date2
+percent    - (default 0.05) percent is the allowed percentage of deviation between offset and compare row counts
+              0.05 (5%) means the difference between the row counts can vary by 5 percent and be accepted
+              a greater difference would send an alert, if compare is not provided this check is not done
+
+Example task 
+(affiliate.publishers table has records from 24h ago, the difference between 24h and 48h row counts should not vary more than 5%): 
+  {"type":"task.dbcheck","info":"db_name?table=affiliate.publishers&compare=48h&percent=0.05"}`
```

Comment:
Can we make this more generic?
"info":"db_name?table=`schema.table`&compare=48h&percent=0.05"

Also the example shows the db_name as the path, but the description shows it as a query param

Reply:
i'll update this

Reply:
I need to add the ability for multiple connections, I didn't use the db_name for anything yet.